frontend/src/export/magicWorkstation.js

```diff
@@ -0,0 +1,23 @@
+/* NOTE think this export option is no longer needed? Magic Workstation doesn't seem to be maintained
+ * Code left here in case that's wrong */
+
+// import {ZONE_MAIN, ZONE_SIDEBOARD} from "../zones";
+
+// export default {
+//   name: "Magic Workstation",
+//   download() {
+//     const arr = [];
+//     [ZONE_MAIN, ZONE_SIDEBOARD].forEach(zoneName => {
+//       const prefix = zoneName === ZONE_SIDEBOARD ? "SB: " : "";
+//       const zone = App.state.gameState.countCardsBy(zoneName, ({setCode, name}) => `${setCode}|${name}`);
+//       Object.entries(zone).forEach(([cardName, count]) => {
+//         const [code, name] = cardName.split("|");
+//         const sanitizedName = name.replace(" // ", "/");
+//         arr.push(`${prefix}${count} [${code}] ${sanitizedName}`);
+//       });
+//     });
+//     return arr.join("\n");
+//   },
+//   downloadExtension: '.mwdeck'
+//   // copy ??
+// }
```

Comment:
@ZeldaZach would you keep that ? I have no idea if this is kept. My idea would be to delete it.

Reply:
It depends whether anyone still uses mtg workstation. I know I and my friends don't but that's a small sample.

I figure disable it in the ui and see if anyone raises an issue. If they do we can resurrect this. But this is git. We could just make a clear commit "delete mtg workstation export" and the code is still findable

Reply:
I'm also not aware of people still using or mentioning it.

But as said above, even their page is heavily outdated and everything seems unmaintained.
>Is MWS still a thing? Looks like their webpage (http://www.magicworkstation.com) is not updated in over 3 years. Their news log makes me think they don't even support cards after Ixalan?!

I assume people are way more exited about the new (and probably way more relevant) export options than the missing MWS...

The MWS hash should also get removed then. I vote for a separate "Remove MWS" PR.

Reply:
I'll raise an issue with it

Reply:
https://github.com/dr4fters/dr4ft/issues/1434